code/modules/explorer_drone/TODO

```diff
@@ -0,0 +1,10 @@
+Future improvements:
```

Comment:
Minor nitpick, this should probably be `TODO.txt` so VSC displays a proper file icon and such.